modules/apps/frontend-icons/frontend-icons-api/src/main/java/com/liferay/frontend/icons/importer/FrontendIconsImporter.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.icons.importer;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+/**
+ * @author Víctor Galán
+ */
+@ProviderType
+public interface FrontendIconsImporter {
+
+	public void importFrontendIcons(
+			long companyId, String name, String spritemap)
```

Comment:
This is a low level API, I didn't want to expose the model classes `FrontendIconsResource` and `FrontendIconsResourcePack` since there is much to do with them outside the module